azkaban-common/src/main/java/azkaban/executor/ExecutorManager.java

```diff
@@ -1337,14 +1336,6 @@ public void run() {
               }
             }
 
-            for (ExecutableFlow flow : finishedFlows) {
-              if (flow.getScheduleId() >= 0
-                  && flow.getStatus() == Status.SUCCEEDED) {
-                ScheduleStatisticManager.invalidateCache(flow.getScheduleId(),
-                    cacheDir);
```

Comment:
In updaterThread, the original intention of this code was to delete the .cache file for the flow when it succeeded. However, there was some bug and flow.getScheduleId() was always -1 and never updated correctly. So .cache file was never deleted once created. So I removed this code and simplified the logic in updaterThread.